cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1558,4 +1558,13 @@
     <description>Kafka topic to which metadata update notifications are published</description>
   </property>
 
+  <property>
+    <name>metadata.updates.publish.enabled</name>
+    <value>true</value>
+    <description>
+      Determines if metadata updates will be published to an external system (like Apache Kafka). External systems can
```

Comment:
I had it so, because ``MetadataChangePublisher`` is an interface and kafka is one implementation.  I will update the comment so its not confusing.